### DISCUSSION TITLE
429 Client Error: Too Many Requests for url

### DISCUSSION THREAD

**Discussion by NinoNinov** [ranaroussi/yfinance#2154] [Q&A]
Hello,

based on that code I aim extracting data for stocks/tickers:
```
list_stocks=[]

for index, value in enumerate (data_2000):
    stock = yf.Ticker(data_2000[index])
    stock_i=stock.info
    stock_i=pd.DataFrame.from_dict(stock_i.items())
    stock_i.columns =['Name', 'Value'] 
    stock_i.rename(columns = {'Value':data_2000[index]}, inplace = True)
    list_stocks.append(stock_i)
```
    

last days for some tickers/stocks receive issue - 429 Client Error: Too Many Requests for url: https://query2.finance.yahoo.com/v10/finance/quoteSummary/TDC?modules=financialData%2CquoteType%2CdefaultKeyStatistics%2CassetProfile%2CsummaryDetail&corsDomain=finance.yahoo.com&formatted=false&symbol=TDC&crumb=Edge%3A+Too+Many+Requests

could soomeone that fixed and download info again for all tickers

Best Regards,

Nino

**Comment by GabrielTehozol** [comment]
I have the same issue, but I don't make too many requests.
my problem goes up when I use the .info.get(), for example like this:
yf.Ticker('NVDA').info.get('marketCap', None)
Edit: I also use a rate limiter, and a time sleep, but even with just one ticker I got the issue.
Thanks in advance.
